src/convertToObject.js

```diff
@@ -3,16 +3,36 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS
+ * properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrSourse = sourceString.split('\n');
+  const convertedObj = {};
+
+  for (let i = 0; i < arrSourse.length; i++) {
+    const element = arrSourse[i].replace(';', '').trim();
+
+    if (element === '' || element === ';') {
+      continue;
+    }
+
+    const part = element.split(':');
```

Comment:
it's enough to do like this 
```suggestion
  const [key, value] = arrSourse[i].split(':');
```